src/components/checkbox/checkbox.ts

```diff
@@ -42,6 +42,12 @@ enum TransitionCheckState {
   Indeterminate
 }
 
+// A simple change event emitted by the MdCheckbox component.
+export class MdCheckboxChange {
+  source: MdCheckbox;
+  value: boolean;
```

Comment:
I think for checkbox and slide-toggle we'd want the property to be `checked` instead of `value`